### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by gabrielcoder247**
### BudgetTK
- Splash screen

A simple page with the name of your app (yes, you need to choose one), and links to the sign up and log in pages.
- Sign up and log in pages

The user should be able to register in the app with full name, email and password (all mandatory).
The user can log into the app using email and password.
If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).
-  page (catagories page)

When the user logs in, they are presented with the categories page.
For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
- Transactions page

For a given category, the list of transactions is presented, ordered by the most recent.
At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
When the user clicks on the "Back" button (<), the user navigates to the home page.
- "Add a new category" page

The user fills out a form to create a new category, indicating their name and icon (both mandatory).
The user click (or taps) the "Save" button to create the new category, and is taken to the home page on success.
When the user clicks on the "Back" button (<), the user navigates to the home page.
"Add a new transaction" page

- The user fills out a form to create a new transaction with:
name (mandatory)
amount (mandatory)
categories (mandatory at least one)
The user click (or taps) the "Save" button to create the new category, and is taken to the transactions page for that category.
When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.
Testing requirements
Create unit and integration tests for all the most important components of your RoR application.
- Technical requirements
You should use Postgres as your database.
You should use devise for authentication.
You should validate all user input to make sure that anyone with bad intentions cannot compromise your app.
You can use a view template engine of your choice (.erb, .slim, .haml).
The project should be deployed and accessible online.

### CODE REVIEW

File: app/views/groups/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      </div>
+    <%end%>
+  <%else%>
+      <p class="p-2 m-2 h3" >There are no transactions for this category</p>
+  <%end%>
+
+
+<div class="w-100 py-3 px-3 btnn">
+    <%= link_to "Add transaction",  new_group_entity_path( @group), class: "btn btn-outline-light  w-75 p-2"%>
+</div>
```
Comment (DeliceLydia) [inline app/views/groups/show.html.erb:56]: - [ ] Currently, when I am trying to create a transaction in a category it doesn't work please address that

File: README.md [hunk 2]
```diff
@@ -1,25 +1,77 @@
...
+
+- Feel free to check the [issues page](https://github.com/faizi2500/recipe-app/issues).
+
+## Show your support
+
+- Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse for giving us this chance
```
Comment (DeliceLydia) [inline README.md]: - [ ] Please add a link to your recorded video explaining key features you worked on in this project remember:
-  The maximum length of the video is 5 minutes.
- The video contains a demo of the project, highlighting all the project features.
- The video contains a highlight about a piece of the code you find interesting or you are proud of making.

Comment (DeliceLydia) [inline README.md:83]: - [ ] Remember to give credits to the author of the original design  [Gregoire Vella on Behance](https://www.behance.net/gregoirevella)

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @gabrielcoder247, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there! Major highlights for your capstone:
- Well done on testing your app
- No linter errors




### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada:  :champagne:  :confetti_ball: 
Hi  :wave:   :wave: @gabrielcoder247 

200w 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your README file looks very professional :+1: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**